ios/Runner/GeneratedPluginRegistrant.m

```diff
@@ -6,58 +6,9 @@
 
 #import "GeneratedPluginRegistrant.h"
 
-#if __has_include(<firebase_auth/FLTFirebaseAuthPlugin.h>)
```

Comment:
This one is, too, generated and any modifications are lost after rebuild

Reply:
Deleted